<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupListsView</class>
 <widget class="QWidget" name="GroupListsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ConfigObjectListView" name="listView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="addRXGroup">
       <property name="text">
        <string>Add RX Group</string>
       </property>
       <property name="shortcut">
        <string>Alt++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remRXGroup">
       <property name="text">
        <string>Delete RX Group</string>
       </property>
       <property name="shortcut">
        <string>Alt+-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigObjectListView</class>
   <extends>QWidget</extends>
   <header>configobjectlistview.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
